props.theme.dir

```diff
@@ -0,0 +1,46 @@
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+[0;7m[38;2;147;161;161m[48;2;7;54;66mprops                                                         0,0-1          All[2 q[2 q[m[38;2;131;148;150m[48;2;0;43;54m[H                                                                                
+[0;1m[38;2;101;123;131m[48;2;0;43;54m~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+[0;7m[38;2;147;161;161m[48;2;7;54;66mprops                                                         0,0-1          All[m[38;2;131;148;150m[48;2;0;43;54m
+                                                                                [H[34h[?25h
```

Comment:
wow, what is this?

Reply:
aaaaah no idea! Nixed on localhost, will push